app/src/main/java/org/sopt/sample/SignUpActivity.kt

```diff
@@ -0,0 +1,45 @@
+package org.sopt.sample
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.google.android.material.snackbar.Snackbar
+import org.sopt.sample.databinding.ActivitySignUpBinding
+
+class SignUpActivity : AppCompatActivity() {
+
+    private lateinit var binding : ActivitySignUpBinding
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivitySignUpBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        init()
+
+    }
+    fun init(){
+        binding.btnSignupFinish.setOnClickListener {
+            if(binding.etId.text.length < 6 || binding.etId.text.length > 10){
+                Snackbar.make(binding.root, "아이디가 잘못되었습니다", Snackbar.LENGTH_SHORT).setAnchorView(binding.etPw).show()
+                return@setOnClickListener
+            }
+            else{
+                val len = binding.etPw.text.length
+                if(len < 8 || len > 12) {
```

Comment:
```suggestion
                if(len in 8..12) {
```
여기도 8자리, 12자리일 때도 포함해주면 좋을 것 같아요!